include/mgos_ade7953.h

```diff
@@ -92,6 +92,10 @@ bool mgos_ade7953_get_apower(struct mgos_ade7953 *dev, int channel, float *watts
 // If reset is true, resets the accumulator.
 bool mgos_ade7953_get_aenergy(struct mgos_ade7953 *dev, int channel, bool reset, float *wh);
 
+// Write the measured power factor to the *pf pointer.
```

Comment:
Can you specify the unit of pf? I suppose something like W/VA ?

from the fluke article on this -- Power factor (PF) is the ratio of working power, measured in kilowatts (kW), to apparent power, measured in kilovolt amperes (kVA)

Reply:
AFAIK, the power factor is a dimensionless number in the range (-1,1). I don't think W/VA is appropriate.